packages/vm/src/crypto.rs

```diff
@@ -1,31 +1,53 @@
+use ed25519_zebra as ed25519;
+
 use k256::{
     ecdsa::signature::{DigestVerifier, Signature as _}, // traits
     ecdsa::{Signature, VerifyingKey},                   // type aliases
 };
 use sha2::Digest; // trait
 
+use std::convert::{TryFrom, TryInto};
+
 use crate::dummy_sha2;
 use crate::errors::{VmError, VmResult};
 
-pub fn secp256k1_verify(message_hash: &[u8], signature: &[u8], public_key: &[u8]) -> VmResult<()> {
+pub fn secp256k1_verify(
+    message_hash: &[u8],
+    signature_bytes: &[u8],
+    public_key_bytes: &[u8],
+) -> VmResult<()> {
```

Comment:
Why all the renamings to `_bytes`? My understanding so far was that Rust encourages variable shadowing and it is perfectly fine to convert and override a name.

Reply:
Yeah, I thought it was for clarity, but I think you'll right, it's not needed.